go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/emirpasic/gods v0.0.0-20190624094223-e689965507ab
 	github.com/fatih/color v1.9.0
 	github.com/go-sql-driver/mysql v1.5.0
-	github.com/gocql/gocql v0.0.0-20200515162754-0714040f3e35
+	github.com/gocql/gocql v0.0.0-20171220143535-56a164ee9f31
```

Comment:
:)

Reply:
Oh yeah. It was a mistake. Luckly, I recalled you told me about it and reverted.